src/lib/viewers/BaseViewer.js

```diff
@@ -1400,6 +1400,30 @@ class BaseViewer extends EventEmitter {
     featureEnabled(feature) {
         return isFeatureEnabled(this.options.features, feature);
     }
+
+    /**
+     * Hides the preview mask element if it exists
+     *
+     * @return {void}
+     */
+    hidePreviewMask() {
+        const previewMask = document.getElementsByClassName('bcpr-PreviewMask')[0];
```

Comment:
Should we short circuit? `document.getElementsByClassName('bcpr-PreviewMask')?[0]`